PythonScripts/ClipPDF.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# importing the required modules 
+import PyPDF2 
+import os
+from os import listdir
+from os.path import isfile, join
+
+#Clip PDFs to size 300kb
+def clipPDF(pdf): 
```

Comment:
Function naming uses underscore in python

Reply:
Can rename pdf to pdf_file_path?

Reply:
done